staging/src/k8s.io/component-base/metrics/BUILD

```diff
@@ -88,14 +88,6 @@ package_group(
     packages = [
         "//cluster/images/etcd-version-monitor",
         "//pkg/controller/endpointslice",
-        "//pkg/controller/volume/attachdetach/metrics",
-        "//pkg/controller/volume/persistentvolume/metrics",
-        "//pkg/kubelet/metrics/collectors",
-        "//pkg/kubelet/pleg",
-        "//pkg/kubelet/pluginmanager/metrics",
-        "//pkg/kubelet/server",
-        "//pkg/kubelet/server/stats",
-        "//pkg/kubelet/volumemanager/metrics",
```

Comment:
Does this mean all the removed packages here have no reference to prometheus now?